### PR TITLE
Feat header title builder

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

File: lib/src/widgets/calendar_header.dart [hunk 1]
```diff
@@ -56,7 +58,7 @@ class CalendarHeader extends StatelessWidget {
               padding: headerStyle.leftChevronPadding,
             ),
           Expanded(
-            child: GestureDetector(
+            child: headerTitleBuilder != null ? headerTitleBuilder : GestureDetector(
```
Comment (aleksanderwozniak) [inline lib/src/widgets/calendar_header.dart:61]: Let's change this line to `headerTitleBuilder?.call(context, focusedMonth) ??`, as the line above results in runtime error (missing params).

**Review by aleksanderwozniak — CHANGES_REQUESTED** [review]
Looks good besides one thing, check the comment I've added.

File: lib/src/widgets/calendar_header.dart (hunk 1 above)
Comment (lohanidamodar) [reply lib/src/widgets/calendar_header.dart:61]: Understood

Comment (lohanidamodar) [reply lib/src/widgets/calendar_header.dart]: done!

**Review by aleksanderwozniak — APPROVED** [review]
Looks great! Thanks for contributing.

**Pull request merged**